src/assets/image/icons/connections/index.ts

```diff
@@ -34,3 +34,5 @@ export { default as LinearIcon } from "@assets/image/icons/connections/linear.sv
 export { default as SalesforceIcon } from "@assets/image/icons/connections/Salesforce.svg?react";
 // Taken from: https://www.svgrepo.com/svg/452111/teams
 export { default as MicrosoftTeamsIcon } from "@assets/image/icons/connections/MicrosoftTeams.svg?react";
+// Taken from: https://iconduck.com/icons/94534/kubernetes
```

Comment:
Just a heads-up: it’s not an MIT license, but I checked and it looks like [CC0](https://creativecommons.org/publicdomain/zero/1.0/deed.en) is okay for both personal and commercial use, so we’re all set! 💪🏼